be/src/exprs/jit/jit_functions.cpp

```diff
@@ -258,16 +259,16 @@ Status JITFunction::llvm_function(JITScalarFunction jit_function, const Columns&
     jit_columns.reserve(columns.size());
     // Extract data and null_data pointers from columns to generate JIT columns.
     for (const auto& column : columns) {
-        auto data_column = FunctionHelper::get_data_column_of_nullable(column);
+        auto data_column = ColumnHelper::get_data_column(column.get());
         auto datums = reinterpret_cast<const int8_t*>(data_column->raw_data());
 
         const int8_t* null_flags = nullptr;
-        if (column->is_nullable()) {
+        if (!column->is_constant() && column->is_nullable()) {
             null_flags = reinterpret_cast<const int8_t*>(
                     ColumnHelper::as_raw_column<NullableColumn>(column)->null_column()->raw_data());
         }
```

Comment:
there are 4 types of columns:
1. constant column; 
2. constant nullable column;
3. nullable column;
4. column;
it seems only type-4 and type-3 are considerated, what if type-1 and type-2 column?

Reply:
here only get null map for nullable columns